src/main/java/teammates/ui/pagedata/AdminActivityLogPageData.java

```diff
@@ -148,99 +157,73 @@ public void generateQueryParameters(String query) {
     }
 
     /**
-     * check current log entry should be excluded as rubbish logs
-     * returns false if the logEntry is regarded as rubbish
+     * @returns true if the current log entry should be included
```

Comment:
Should be `* Returns true ...` ?
Overview statement takes priority over `@return`